win32/tests/threadpool_win32_ut/threadpool_win32_ut.c

```diff
@@ -1066,10 +1073,12 @@ TEST_FUNCTION(threadpool_timer_start_fails_when_underlying_functions_fail)
     filetime_expected.dwLowDateTime = ularge_due_time.LowPart;
 
     STRICT_EXPECTED_CALL(malloc(IGNORED_ARG));
+    STRICT_EXPECTED_CALL(interlocked_exchange(IGNORED_ARG, 1)).CallCannotFail();

```

Comment:
>STRICT_EXPECTED_CALL(interlocked_exchange(IGNORED_ARG, 1)).CallCannotFail(); [](http://example.com/codeflow?start=4&length=76)

I would add a comment next to the lines that do strict expected call for this interlocked_exchange(IGNORED_ARG, 1), saying that it is the reference count from THANDLE.

Reply:
In all places in this file of course.

Reply:
added here and all places